Hazel/src/Hazel/Debug/Instrumentor.h

```diff
@@ -1,129 +1,116 @@
 #pragma once
 
-#include <string>
-#include <chrono>
 #include <algorithm>
+#include <chrono>
 #include <fstream>
+#include <string>
 
 #include <thread>
 
 namespace Hazel {
-	struct ProfileResult
```

Comment:
In this whole file, please convert this back to tabs instead of spaces.

Reply:
Resolved in https://github.com/TheCherno/Hazel/pull/174/commits/3aff8a00f063efb429d5e0e766820b9df9243207